core/domain/exp_jobs_one_off.py

```diff
@@ -470,3 +471,115 @@ def map(item):
     @staticmethod
     def reduce(key, values):
         yield (key, values)
+
+
+class ExplorationContentValidationJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that checks the html content of an exploration and validates it.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+
+        def find(key, dictionary):
```

Comment:
I'm not too familiar with the backend codebase, but:
1) Is there a similar function somewhere else in our codebase which we can reuse?
2) If the answer to 1) is false, can we put this function in a different place, rather than as a nested fn?

Reply:
I didn't get something similar. I shifted the function to `exp_services.py`.
Done!